bin/pycbc_hdf5_splitbank

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2016  Soumi De
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+"""
+The code reads in a compressed template bank and splits it up into
+smaller banks where the number of smaller banks is a user input
+"""
+
+import argparse
+import h5py
+import logging
+import pycbc, pycbc.version
+from pycbc.waveform import bank
+__author__  = "Soumi De <soumi.de@ligo.org>"
+__version__ = pycbc.version.git_verbose_msg
+__date__    = pycbc.version.date
+__program__ = "pycbc_hdf5_splitbank"
+
+
+parser = argparse.ArgumentParser(description=__doc__[1:])
+parser.add_argument("--bank-file", type=str,
+                    help="Bank hdf file to load.")
+parser.add_argument('--templates-per-bank',
+                    help='number of templates in the output banks', type=int)
+parser.add_argument("--number-of-banks", type=int,
+                    help="Number of banks")
```

Comment:
Please add some more description to the help message here, making it clear that either `templates-per-bank` or `number-of-banks` can be specified, not both. As is, it's confusing as to why both options are there. Also, since these are mutually exclusive, add a check below that will raise an error if both options are specified, or if neither option is specified.